doc/usermanual/lighttable/panels/styles.xml

```diff
@@ -74,6 +74,11 @@
         history stack items which you want to not be part of the newly created style.
       </para>
 
+      <para>
+          The panel supports a hierarchical view, this means you can create categories using a
+          | as a sperator. For example "print|tone curve +0.5 EV" will create a print category.
```

Comment:
typo: 'sperator', I propose:

...using the pipe symbol <quote>|</quote> as a separator. For example <quote>...</quote> will create...

Reply:
Fixed, thanks!